integration/targets_test.go

```diff
@@ -113,7 +113,7 @@ func testTargets(t *testing.T, context spec.G, it spec.S) {
 			var logs fmt.Stringer
 			image, logs, err = pack.Build.
 				WithPullPolicy("never").
-				WithEnv(map[string]string{"BP_GO_TARGETS": "first:./second"}).
```

Comment:
I think we can reduce the number of `pack build` calls in these integration tests if we simply add a second container to the test that starts on 67. That'll also avoid the identical-images problem.